src/plugins/dataflow/model/dataflow-program-model.ts

```diff
@@ -46,6 +46,7 @@ export interface DataflowSocketSnapshotOut extends SnapshotOut<typeof SocketMode
 const DataflowNodeDataModel = types.
   model("DataflowNodeData", {
     plot: types.maybe(types.boolean),
+    displayNameInsertionOrder: types.maybe(types.string),
```

Comment:
Defining this in the model made it so that when it was in `ProgramMode = Done` , the `DataflowNode` module re-rendered the `displayNameInsertionOrder`.